packages/plugin-core/assets/dendron-ws/tutorial/treatments/main/tutorial.conclusion.md

```diff
@@ -15,6 +15,8 @@ With fast lookup, flexible hierarchies, note linking, and rich formatting, you'r
 
 From here, you can start adding your notes to this workspace or, if you'd like to start fresh, you can create a new workspace for your notes with the `Dendron: Initialize Workspace` Command.
 
+> Coming from Obsidian? Click [here](command:dendron.importObsidianPod) to import your Obsidian notes (or any markdown notes) into Dendron to see how they look.
```

Comment:
right now we copy/paste the tutorial from the website. mixing in command uris here will add more upkeep. fine for now but lets come up with a plan to keep the tutorial here and the website in sync